hubploy/auth.py

```diff
@@ -216,32 +232,45 @@ def cluster_auth_gcloud(deployment, project, cluster, zone, service_key):
     yield
 
 
-def cluster_auth_aws(deployment, project, cluster, zone, service_key):
+def cluster_auth_aws(deployment, project, cluster, zone, service_key = None, eks_role = None):
```

Comment:
Let's call this parameter 'role' rather than 'eks_role'? The cluster section is already eks specific.